vm/starnix/starnix.go

```diff
@@ -187,8 +215,9 @@ func (inst *instance) Close() error {
 }
 
 func (inst *instance) startFuchsiaVM() error {
-	err := inst.ffx("emu", "start", "--headless", "--name", inst.name, "--net", "user")
-	if err != nil {
+	inst.runFfx(1*time.Minute, "config", "get", "product.path")
```

Comment:
Nit: any reason the timeout is different from the same command in `pool.Create()`?

Reply:
Nope! :) Thanks for catching.